src/docs/user/references/Variables.adoc

```diff
@@ -0,0 +1,487 @@
+
+Variables can come at hand while developing ProActive applications.
+Propagate data between tasks, customize jobs, and store useful debug information are a few examples of
+how variables can ease development. ProActive has basically two groups of variables :
+
+- <<_workflow_variables>>, declared on the XML job definition.
+- <<_dynamic_variables>>, system variables or variables created at task runtime.
+
+[[_workflow_variables]]
+==== Workflow variables
+
+Workflow variables are declared in the XML job definition. Within the definition we can have variables in several
+levels, at job level the variables are visible and shared by all tasks. At task level, variables are visible
+only within the task execution context.
+
+[[_job_variables]]
+===== Job variables
+
+In a workflow you can define *job variables* that are shared and visible by all tasks.
+
+A job variable is defined using the following attributes:
+
+* *name*: the name of the variable
+* *value*: the value of the variable
+* *model*: the type of the value (optional). See section <<_variable_model,Variable Model>>
+
+
+[[_templating]]
+Variables are very useful to use workflows as templates where only a few parameters change for each
+execution. Upon submission you can define variable values either from <<_scheduler_command_line, CLI>> or from
+the Web interfaces.
```

Comment:
Let's say from the **ProActive Workflow Studio** instead of Web interfaces

Reply:
It is also possible to define variables when submitting workflows from the scheduler portal (and as well in the rest api)